SetupGuide_Plugin_Admin.php

```diff
@@ -94,8 +94,8 @@ public function skip() {
 	 * @uses $_POST['choice'] TOS choice: accept/decline.
 	 */
 	public function set_tos_choice() {
-		if ( wp_verify_nonce( $_POST['_wpnonce'], 'w3tc_wizard' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
-			$choice          = empty( $_POST['choice'] ) ? null : sanitize_key( $_POST['choice'] );
+		if ( wp_verify_nonce( Util_Request::get_string( '_wpnonce' ), 'w3tc_wizard' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
+			$choice          = empty( Util_Request::get_string( 'choice' ) ) ? null : Util_Request::get_string( 'choice' );
```

Comment:
You can just use this:
```
$choice          = Util_Request::get_string( 'choice' );
```
The code blow checks against the allowed choices.  If it's empty, etc., then it will have the same result; `wp_send_json_success();`

Reply:
Simplified assignment code.